go.mod

```diff
@@ -4,24 +4,24 @@ go 1.16
 
 require (
 	github.com/csi-addons/volume-replication-operator v0.2.0
-	github.com/kube-object-storage/lib-bucket-provisioner v0.0.0-20210311161930-4bea5edaff58
+	github.com/kube-object-storage/lib-bucket-provisioner v0.0.0-20210818162813-3eee31c01875
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.15.0
 	github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
-	github.com/openshift/library-go v0.0.0-20210727084322-8a96c0a97c06
+	github.com/openshift/library-go v0.0.0-20210916194400-ae21aab32431
 	github.com/openshift/ocs-operator v0.0.1-alpha2
-	github.com/ramendr/ramen v0.0.0-20220201125512-ff29bf0d3dbb
-	github.com/rook/rook v1.7.0
+	github.com/ramendr/ramen v0.0.0-20220314215400-90d1d91e4875
+	github.com/rook/rook v1.7.3-0.20210830134613-f2b966caf356
 	github.com/spf13/cobra v1.1.3
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	k8s.io/api v0.22.1
-	k8s.io/apimachinery v0.22.1
+	k8s.io/api v0.22.6
```

Comment:
bumping up ramen is increasing k8 version but in replace we are pointing old version of k8 already